src/main/java/org/tron/common/runtime/vm/VM.java

```diff
@@ -78,6 +78,13 @@ public void step(Program program) {
         throw Program.Exception.invalidOpCode(program.getCurrentOp());
       }
 
+      // hard fork for 3.2
+      if (false) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this if statement. ([squid:S1145](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1145))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1690&issues=AWbtJhEzTJjFgJLTBV4x&open=AWbtJhEzTJjFgJLTBV4x)

[ISSUE_KEY]: AWbtJhEzTJjFgJLTBV4x